.github/workflows/build_and_archive_release_docs.yml

```diff
@@ -0,0 +1,96 @@
+name: Build and archive documentation from release branches
+
+on:
+  push:
+    branches:
+      - 'uyuni-20[0-9][0-9].[0-9][0-9]' # Only master will build both Uyuni and SUSE Manager documentations
+      - 'manager-4.3-MU-4.3.[0-9][0-9]?' # Other branches will only build SUSE Manager documentation
```

Comment:
We need to define the format for alpha/beta/rc/gmc branches

Reply:
I suggest:
```
Manager-5.0-[Alpha|Beta|RC|GMC|GM]*
```

Reply:
In case it is case-sensitive, I'd like to go for "alpha|beta".  The rest can stay.

Reply:
Done, I added:
```
      - 'manager-5.0-[alpha|beta|rc|gmc|gm]*' # Other branches will only build SUSE Manager documentation

Reply:
Didn't fly, so I had to create and merge https://github.com/uyuni-project/uyuni-docs/pull/2794